### PR TITLE
switch to ocf.berkeley.edu

### LINKED ISSUE #1: Set up federation
Currently, the current server namespace is `matrix.ocf.berkeley.edu`, which means entities in the Matrix world are addressed by `matrix.ocf.berkeley.edu`. In the future, we should move this to `ocf.berkeley.edu`. The following has to occur:

- [x] Update `server_name` to `ocf.berkeley.edu`
- [x] Serve a file under `.well-known/matrix/server` serving according to [docs/delegate.md](https://github.com/matrix-org/synapse/blob/master/docs/delegate.md). See [this](https://github.com/ocf/puppet/blob/be68a0efe0d1c501cc386ea69682441a512b2c6c/modules/ocf_www/manifests/site/www.pp#L46-L49) for Mastodon example.

Federation tester: https://matrix.org/federationtester

### PR DESCRIPTION
**Pull request by nikhiljha**
fixes #1 
depends on https://github.com/ocf/puppet/pull/953

### CODE REVIEW

File: homeserver.yaml [hunk 1]
```diff
@@ -1121,7 +1121,7 @@ form_secret: "loG32N*yCgr;Pp.Ih1l_fuw#G:0ii@D8J0cuPbed#-o25:sKLX"
 
 # Path to the signing key to sign messages with
 #
-signing_key_path: "/data/matrix.ocf.berkeley.edu.signing.key"
+signing_key_path: "/data/ocf.berkeley.edu.signing.key"
```
Comment (encadyma) [inline homeserver.yaml:1124]: Not sure if changing this might break

Comment (nikhiljha) [reply homeserver.yaml:1124]: I think we want to break it, switching the homeserver name requires a key + database reset or other servers will get confused.

**Comment by nikhiljha** [comment]
Oh, and we should also probably fix secrets before merging this, because merging this == "production"

**Comment by encadyma** [comment]
I can get to moving the secrets out in #7

**Comment by cg505** [comment]
yeah, this will probably have to be rebased and carefully rolled out

**Comment by nikhiljha** [comment]
plan is to move all this into kmo's branch

**Pull request closed without merging**